applications/software-libraries.md

```diff
@@ -0,0 +1,59 @@
+# Software Libraries
+
+The following describes the approach we adopt when making use of external software libraries in our applications.
+
+## Centrally capture your application's dependencies *in code*
+
```

Comment:
I think that might have gone too far. I like the clarity about remote and external services, but I'm not sure all the technologies record the dependencies that deep.

In Ruby we write the `Gemfile`, but `Gemfile.lock` is auto created and maintained by bundler. But in NodeJS and Java (with Maven) I'm not sure you get the equivalent of Ruby's `.lock` file. You can obviously find them once you've grabbed the project and built it locally, but as the title is **Centrally capture your application's dependencies _in code_**, I think the new text no longer fits with that premise.

*Hope that makes sense* 😟